libs/product/src/reducers/configurable-product-entities/configurable-product-entities.reducer.ts

```diff
@@ -47,36 +48,75 @@ export function daffConfigurableProductEntitiesReducer<T extends DaffProduct, V
 			delete upsertedEntity[action.attributeId];
```

Comment:
I don't think you need these first 2 lines. They don't make sense with the array-style attributes.

Reply:
Yea, I think that was from the dictionary implementation. removed